playbooks/ansible/roles/sit.cephfs/templates/smb_share.conf.j2

```diff
@@ -12,3 +12,8 @@ path = {{ subvol }}
 {%- else +%}
 path = {{ path }}
 {%- endif %}
```

Comment:
In this case it doesn't happen, but without the `+` we may lose the new line character to separate options in different lines.

```suggestion
{%- endif +%}
```